### PR TITLE
CLOUD-3470 Exclude Galleon layers during s2i

### PR DESCRIPTION
**Pull request by jfdenise**

### CODE REVIEW

File: openshift/EAP7-1378_Exclude_Galleon_layers.adoc [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+* datasources-web-server: _datasources_ layer can be excluded
+- _GALLEON_PROVISION_LAYERS=datasources-web-server,-datasources_
+
+* jaxrs-server: _jpa_ and _jpa,datasources_ can be excluded. NB: Excluding _datasources_ alone is not supported, _jpa_ requires it.
+- _GALLEON_PROVISION_LAYERS=jaxrs-server,-jpa_
+- _GALLEON_PROVISION_LAYERS=jaxrs-server,-jpa,-datasources_
+
+* cloud-server:  Supports the exclusions of _jaxrs-server_. In addition _observability_ and _open-tracing_ layers can be excluded
+- _GALLEON_PROVISION_LAYERS=_cloud-server,-observability,[jaxrs-server exclusions]_
```
Comment (jmesnil) [inline openshift/EAP7-1378_Exclude_Galleon_layers.adoc]: preview of the file is not correct. It is confusing to get `_` in the output (I assume that

Could you please use a code block to make sure there is no confusion on the syntax?

**Comment by darranl** [comment]
Is there any community issue that can be used as the overall tracker instead of an internal issue the community can not access?

**Comment by jfdenise** [comment]
@darranl , yes we have https://issues.redhat.com/browse/CLOUD-3470 Will rename the file name.

File: openshift/CLOUD-3470_Exclude_Galleon_layers.adoc [hunk 2]
```diff
@@ -0,0 +1,116 @@
+= CLOUD-3470, Exclude Galleon layers during s2i
+:author:            JF Denise
+:email:             jdenise@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          openshift,galleon,layers
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse/
```
Comment (rhusar) [inline openshift/CLOUD-3470_Exclude_Galleon_layers.adoc:9]: this is now https://issues.redhat.com/browse/

File: openshift/CLOUD-3470_Exclude_Galleon_layers.adoc [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+- ```GALLEON_PROVISION_LAYERS=cloud-server,-observability,[jaxrs-server exclusions]```
+- ```GALLEON_PROVISION_LAYERS=cloud-server,-open-tracing,[jaxrs-server exclusions]```
+- ```GALLEON_PROVISION_LAYERS=cloud-server,-observability,-open-tracing,[jaxrs-server exclusions]``` 
+NB: This leads to an error, _open-tracing_ being already excluded when excluding _observability_.
+
+* observability: _open-tracing_ can be excluded
+- ```GALLEON_PROVISION_LAYERS=<core layer>,observability,-open-tracing```
+
+* microprofile: _microprofile-config_,_microprofile-fault-tolerance_,_microprofile-jwt_,_microprofile-metrics_,_microprofile-openapi_,_open-tracing_ can be excluded
+- ```GALLEON_PROVISION_LAYERS=<core layer>,microprofile,-microprofile-config,-microprofile-fault-tolerance,-microprofile-jwt,-microprofile-metrics,-microprofile-openapi,-open-tracing```
```
Comment (rhusar) [inline openshift/CLOUD-3470_Exclude_Galleon_layers.adoc:83]: the `microprofile` is now named `microprofile-platform`

**Comment by jfdenise** [comment]
This proposal is out of date. Galleon exclusion for WildFly s2i image is already in and documented. Closing it.

**Pull request closed without merging**